api-doc/openapi.yaml

```
@@ -1014,6 +1110,70 @@ components:
         example:
           id: 3a896646-70e0-4943-a3e3-01f29d0541f1
           parameter: kalle tetr
+    Timer:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        name:
+          type: string
+        start_time:
+          type: string
+          format: date-time
+        end_time:
+          type: string
+          format: date-time
+      required:
+        - id
+        - timestamps
```

```suggestion
        - start_time
        - end_time
```
I made name mandatory as well, but can drop that requirement

---

name is now mandatory